networkx/generators/classic.py

```diff
@@ -1,23 +1,19 @@
 """Generators for some classic graphs.
 
-The typical graph generator is called as follows:
+The typical graph builder function is called as follows:
 
 >>> G = nx.complete_graph(100)
 
 returning the complete graph on n nodes labeled 0, .., 99
-as a simple graph. Except for empty_graph, all the generators
+as a simple graph. Except for empty_graph, all the functions
```

Comment:
```suggestion
as a simple graph. Except for `empty_graph`, all the functions
```
Might as well make it a link in the html